frontend/admin/.babelrc

```diff
@@ -2,6 +2,13 @@
   "plugins": [
     "@babel/plugin-proposal-class-properties",
     "@babel/plugin-proposal-private-methods"
+  ],
+  "presets": [
+    ["@babel/preset-env", {
+      "useBuiltIns":"usage",
+      "corejs":3,
```

Comment:
It's [recommended](https://babeljs.io/docs/en/babel-preset-env#corejs) to use minor version numbers. That link also mentions options for polyfilling [proposals](https://github.com/zloirock/core-js#ecmascript-proposals), but I don't think we're using any of them?

Reply:
Good call.  I set it to 3.21 since that's what npm put in the package.json.  I don't know of us using proposals but I'm hazy on the topic.